src/app/config.py

```diff
@@ -29,3 +29,4 @@
         "Missing Credentials. Please set 'SUPERUSER_LOGIN' and 'SUPERUSER_PWD' in your environment variables")
 
 BUCKET_NAME: str = "mypyroneartest"
+DUMMY_BUCKET_FILE = "app/static/dummy_media.jpg"
```

Comment:
This is typically something we could put in a release attachment, I'm really against making the github history heavier with a content file, if this can be avoided. And since this is for testing, we can even get the URL of a publicly available image, which might be the best option here

Reply:
Here is one for instance: https://ec.europa.eu/jrc/sites/jrcsh/files/styles/normal-responsive/public/growing-risk-future-wildfires_adobestock_199370851.jpeg

Reply:
Ok but I think we can't just return a URL, I understood from @akilditu comment in https://github.com/pyronear/pyro-api/issues/61 that the API should return an image directly.
 Downloading that file everytime and then returns it seems heavy to me.

Reply:
Mmmh why not let the API return the URL, and have the client download from the URL? (in case it's something that requires creds, we pass parameters down in the URL?)

That would remove a lot of data transit through the API

Reply:
I have created 3 methods in the client for the different use case. We will delete what is not required anymore.